app/controllers/contestants_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ContestantsController < ApplicationController
+  def index
+    @contestants = Contestant.all
+  end
+
+  def new
+    @project = Project.find(params[:id])
+  end
+
+  def create
```

Comment:
This looks great. This may also be better served in a `ContestantProjectsController` to split up the responsibility a bit.